File: test/unit/code_parser/whole_tests/views/mer_2010.php
<?php
// Copyright (c) 2000 ActivePDF, Inc.
// ActivePDF Meridian 2010
// Example generated 01/01/2000 
?>

<?php
// Instantiate Object
$oMER = new COM("APMeridian.Object");

$intWait = $oMER->Wait(30);
if ($intWait != 0) {
  Error("Wait", $intWait);
}

// Release Object
$oMER = null;

// Process Complete
echo "Done!";

// Error Handling
function Error($method, $outputCode) {
  echo "Error in " . $method . ": " . $outputCode;
}
?>

File: test/unit/code_parser/whole_tests/views/mer_2010.rb
# Copyright (c) 2000 ActivePDF, Inc.
# ActivePDF Meridian 2010
# Example generated 01/01/2000 

require 'win32ole'

# Instantiate Object
oMER = WIN32OLE.new("APMeridian.Object")

intWait = oMER.Wait(30)
if intWait != 0
  puts "Error in Wait: #{intWait}"
end

# Release Object
oMER = ''

# Process Complete
puts "Done!"